packages/bun-vscode/src/features/tests/index.ts

```diff
@@ -101,9 +102,20 @@ export function registerTestCodeLens(context: vscode.ExtensionContext) {
   );
 }
 
+const outputChannel = vscode.window.createOutputChannel("Bun - Test Results", {
+  log: true,
+});
+
+const appendLines = (chunk: { toString(): string }) => {
+  const lines = chunk.toString().split("\n");
```

Comment:
this will cause some issues with non-ascii characters

instead we should set encoding to utf8 on the stderr/stdout and not call .toString

Reply:
The reason for the `toString()` was so we can split by lines, as vscode prepends a timestamp to every log. Splitting them meant we could have every log lined horizontally up

Reply:
Splitting is fine. The issue is these are Buffer objects which means it can be an incomplete unicode sequence. Using string_decoder will prevent partially written unicode from appearing in terminal output. Settings the encoding on the stdout & stderr will fix it by only returning already decoded unicode